frontend/src/components/SystemsTable/index.tsx

```diff
@@ -197,7 +242,7 @@ export function SystemsTable({
       />
       <AnalysisDrawer
         systems={systems.filter((sys) =>
-          activeSystemIDs.includes(sys.system_id)
+          activeSystemIDArray.includes(sys.system_id)
```

Comment:
There's one edge case that is not handled here. If the user provides a system ID that is not in `systems`, that ID is simply ignored which can be confusing. There are two scenarios when this can happen:
1. The ID is not valid. e.g. The user pasted a wrong system ID. I think this is less important because it's less likely to happen but it'll be nice if we can show an error message.
2. The ID is valid but not on the current view. This can happen when a user shares a URL with private systems. Or when the system is simply not on the first page. This is a more common scenario and needs to be handled.

To fix this, we probably need to update the backend. I think we can remove support for activeSystemID in the URL for this PR and add it back in another PR. What do you think?

Reply:
This has been removed for the PR. Active systems now have nothing to do with the URL. 
It is already included in a `SystemFilter` object, so whenever we have the backend updated and want to turn this feature on, we can change `SystemTable` to use the `SystemFilter`'s activeSystemIDs instead of the states defined separately.